crates/forge-runner/src/sierra_casm_runner.rs

```diff
@@ -380,6 +380,7 @@ impl SierraCasmRunner {
 
     /// Returns the initial value for the gas counter.
     /// If `available_gas` is None returns 0.
+    #[allow(dead_code)]
```

Comment:
Why is it needed here?

Reply:
I've removed the initial_gas calculation because the current way was not appropriate.
It mixed old and new gas logic